mm/2s2h/BenGui/SearchableMenuItems.h

```diff
@@ -0,0 +1,1514 @@
+            if (UIWidgets::SliderFloat(widget.widgetName, std::get<float*>(widget.widgetOptions.valuePointer), floatMin,
+                                       floatMax,
+                                       {
+                                           .color = menuTheme[menuThemeIndex],
+                                           .tooltip = widget.widgetTooltip,
+                                           .disabled = disabledValue,
+                                           .disabledTooltip = disabledTooltip,
+                                       })) {
+                if (widget.widgetCallback != nullptr) {
+                    widget.widgetCallback(widget);
+                }
+            }
+        } break;
+        case WIDGET_CVAR_SLIDER_INT:
+            if (UIWidgets::CVarSliderInt(
+                    widget.widgetName, widget.widgetCVar, std::get<int32_t>(widget.widgetOptions.min),
+                    std::get<int32_t>(widget.widgetOptions.max), std::get<int32_t>(widget.widgetOptions.defaultVariant),
+                    {
+                        .color = menuTheme[menuThemeIndex],
+                        .tooltip = widget.widgetTooltip,
+                        .disabled = disabledValue,
+                        .disabledTooltip = disabledTooltip,
+                    })) {
+                if (widget.widgetCallback != nullptr) {
+                    widget.widgetCallback(widget);
+                }
+            };
+            break;
+        case WIDGET_CVAR_SLIDER_FLOAT: {
+            float floatMin = (std::get<float>(widget.widgetOptions.min) / 100);
+            float floatMax = (std::get<float>(widget.widgetOptions.max) / 100);
+            float floatDefault = (std::get<float>(widget.widgetOptions.defaultVariant) / 100);
+            if (UIWidgets::CVarSliderFloat(widget.widgetName, widget.widgetCVar, floatMin, floatMax, floatDefault,
+                                           {
+                                               .color = menuTheme[menuThemeIndex],
+                                               .tooltip = widget.widgetTooltip,
+                                               .disabled = disabledValue,
+                                               .disabledTooltip = disabledTooltip,
+                                           })) {
+                if (widget.widgetCallback != nullptr) {
+                    widget.widgetCallback(widget);
+                }
+            }
+        } break;
+        case WIDGET_BUTTON:
+            if (UIWidgets::Button(widget.widgetName, { menuTheme[menuThemeIndex], widget.widgetOptions.size,
+                                                       widget.widgetTooltip, disabledValue, disabledTooltip })) {
+                if (widget.widgetCallback != nullptr) {
+                    widget.widgetCallback(widget);
+                }
+            }
+            break;
+        case WIDGET_WINDOW_BUTTON: {
+            if (widget.widgetOptions.windowName == "") {
```

Comment:
`result of comparison against a string literal is unspecified (use an explicit string comparison function instead) [-Wstring-compare]`

`strlen` or this to be safer

```suggestion
            if (widget.widgetOptions.windowName == nullptr || widget.widgetOptions.windowName[0] == '\0') {
```